recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
     patches:
         - clang.patch  # [osx]
         - prepend-dll.patch  # [win]
+        - disable_jpeg12.patch  # [win]
```

Comment:
> @ocefpaf Shall we wait for the CI's to finish and add that comment just before merging this?

Yes. And then you can commit with `[skip ci]` since there are no changes to the build.